portal-web/test/functional/com/liferay/portalweb/dependencies/groovy-script-portal-cache-put-modules.groovy

```diff
@@ -18,4 +18,4 @@ try {
 }
 catch (Exception e) {
 	out.println(e);
-}
\ No newline at end of file
+}
```

Comment:
@dantewang 
Why this is needed? It seems just adding an empty line, is this requested by SF?

Tina